docs/lua/callbacks.md

```diff
@@ -13,4 +13,23 @@ end
 ```
 
 * `onClientDisconnected(event)` - Called when a client is disconnected from the server
-	* event - [client_connection_info](classes/client_connection_info.md)
+	* event:[client_connection_info](classes/client_connection_info.md)
+* `onClientJoin(client)` - Called when a client (user) is entering the world such as after login
+	* client:[ClientConnection](classes/ClientConnection.md)
+* `onClientLeave(client)` - Called when a client (user) is leaving the world such as after logout
+	* client:[ClientConnection](classes/ClientConnection.md)
+* `onUserLogin(client, user)` - Called when a user is logged in. **ASSIGN A USER BEFORE RETURNING**
+	* client:[ClientConnection](classes/ClientConnection.md)
+	* user:[User](classes/User.md)
+
+```lua
+function onUserLogin(client, user)
+	user_list = save.user_list
+	user = user_list:FindUser(username)
+	client.user = user
+end
+```
```

Comment:
the argument in this example should be `username` not `user`
Also the save object doesn't have a doc page, even if it is a stub/wip for now.

Reply:
https://github.com/trillek-team/tec/issues/214